server/api.js

```diff
@@ -18,9 +18,9 @@ router.get("/clients", (req, res) => {
 const saveUser = async (element) => {
 	return new Promise((resolve, reject) => {
 		const newClientDateAdded = element.date_added;
-		if (!newClientDateAdded || isNaN(!newClientDateAdded)) {
```

Comment:
It wasn't a completely bad idea, but it should have been performed over the result of Date.parse() rather than a string.